src/components/WebexAvatar/WebexAvatar.js

```diff
@@ -1,6 +1,34 @@
-import React from 'react';
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
 import {Avatar} from '@momentum-ui/react';
 
-export default function WebexAvatar(props) {
-  return <Avatar {...props} />;
+const propTypes = {
+  personID: PropTypes.string.isRequired,
+  adapter: PropTypes.object.isRequired,
+};
+
+export default class WebexAvatar extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.props = props;
+    this.state = {
+      person: {},
+    };
+  }
+
+  componentDidMount() {
+    const onNext = (person) => this.setState({person});
+    // eslint-disable-next-line no-console
+    const onError = (error) => console.error(error.message);
+
+    this.props.adapter.getPerson(this.props.personID).subscribe(onNext, onError);
+  }
+
+  render() {
+    const {avatar, displayName, status} = this.state.person;
+
+    return <Avatar src={avatar} title={displayName} type={status} alt={displayName} />;
```

Comment:
Does momentum default the alt to title? If so, the `alt={displayName}` is redundant.

Reply:
no it does not: https://github.com/momentum-design/momentum-ui/blob/master/react/src/lib/Avatar/index.js#L238